Examples/CaseStudies/SwiftUICaseStudies/03-Effects-LongLiving.swift

```diff
@@ -56,11 +56,14 @@ extension DependencyValues {
 
 private enum ScreenshotsKey: DependencyKey {
   static let liveValue: @Sendable () async -> AsyncStream<Void> = {
-    AsyncStream(
-      NotificationCenter.default
-        .notifications(named: UIApplication.userDidTakeScreenshotNotification)
-        .map { _ in }
-    )
+    AsyncStream { (continuation: AsyncStream<Void>.Continuation) -> Void in
+      Task {
+        for await _ in NotificationCenter.default.notifications(named: UIApplication.userDidTakeScreenshotNotification) {
+          continuation.yield()
+        }
+        continuation.finish()
+      }
+    }
```

Comment:
We can actually revert this back because we have a fix for this in [another PR](https://github.com/pointfreeco/swift-composable-architecture/pull/3428/files#diff-13a1d222d1818ac29e2d5d564adcf8db804f8bb05f6d33073065fce19fde01abR58) since it is now possible to use `any AsyncSequence<_, _>`.

```suggestion
    AsyncStream(
      NotificationCenter.default
        .notifications(named: UIApplication.userDidTakeScreenshotNotification)
        .map { _ in }
    )
```